README.md

```diff
@@ -1,2 +1,33 @@
-# os2forms_organisation
-os2forms_organisation
+# OS2Forms organisation
+
+## Installation
+
+```sh
+composer require os2forms/os2forms_organisation
+drush pm:enable os2forms_organisation
+```
+
+Edit settings on `/admin/os2forms_organisation/settings`.
+
+## Drush commands
+
+```sh
+drush os2forms-organisation:lookup --help
```

Comment:
The command is actually called `os2forms_organisation:lookup`. Remember to test your documentation by copying and pasting in a terminal.